MediaBrowser.Controller/Syncplay/GroupInfo.cs

```diff
@@ -46,19 +46,19 @@ public class GroupInfo
         public DateTime LastActivity { get; set; }
 
         /// <summary>
-        /// Gets the partecipants.
+        /// Gets the participants.
         /// </summary>
-        /// <value>The partecipants.</value>
-        public readonly ConcurrentDictionary<string, GroupMember> Partecipants =
-        new ConcurrentDictionary<string, GroupMember>(StringComparer.OrdinalIgnoreCase);
+        /// <value>The participants, or members of the group.</value>
+        public readonly ConcurrentDictionary<string, GroupMember> Participants =
```

Comment:
I'm starting to feel wary of data races here as well... example - computing highest ping and in parallel getting a new client joining the group might produce unexpected results.
So it's probably better to do a manual locking here, too.

Reply:
> computing highest ping and in parallel getting a new client joining the group

I don't think that can happen anymore as `SyncplayManager` handles the locking now.
I should probably replace the `ConcurrentDictionary` as it's not really needed.

Shall I add a notice somewhere stating that all operations of `SyncplayController` must happen inside a lock? Or do you consider safer introducing another lock?

Reply:
It's better to avoid nested locking, because doing it correctly is _hard_. Just add a notice that outside locking is needed.